code/game/objects/effects/spiderwebs.dm

```diff
@@ -80,6 +80,10 @@
 	sealed = TRUE
 	can_atmos_pass = ATMOS_PASS_NO
 
+/obj/structure/spider/stickyweb/sealed/Initialize(mapload)
+	. = ..()
+	air_update_turf(TRUE, TRUE)
```

Comment:
use named arguments here, I have no idea what TRUE, TRUE means

Reply:
> use named arguments here, I have no idea what TRUE, TRUE means

It's literary what every object that stops atmos uses the reason sealed webs weren't stoping stuff like plasma is because it didn't have this it has to be TRUE, TRUE
![image](https://github.com/tgstation/tgstation/assets/84478872/3d4e4637-6861-40d2-83c3-ebf37ff146ae)